crates/test-utils/src/deploy_helpers.rs

```diff
@@ -634,8 +634,19 @@ impl DeployedKakarot {
     }
 }
 
+/// Returns the dumped Katana state with deployed Kakarot + EVM contracts.
+pub fn dumped_kakarot_state() -> SerializableState {
+    // Get dump path
+    let path = root_project_path!(".katana/dump.json");
+
+    // Create SerializableState from dumped state
+    let state = std::fs::read_to_string(path).expect("Failed to read Katana dump");
+    serde_json::from_str(&state).expect("Failed to deserialize Katana dump")
+}
+
 /// Returns a `StarknetConfig` instance customized for Kakarot.
-pub fn kakarot_starknet_config() -> StarknetConfig {
+/// If `with_dumped_state` is true, the config will be initialized with the dumped state.
+pub fn kakarot_starknet_config(with_dumped_state: bool) -> StarknetConfig {
     let kakarot_steps = std::u32::MAX;
```

Comment:
in which sense are these _kakarot_ steps?

Reply:
I think it would be better to rename this to `max_steps`